transformations/chinese_person_named_entities_gender/test.json

```diff
@@ -0,0 +1,55 @@
+{
+  "type": "chinese_person_named_entities_gender",
+  "test_cases": [
+    {
+      "class": "ChinesePersonNamedEntitiesAndGender",
+      "inputs": {
+        "sentence": "邹志元给自己所有的衬衣都缝上了标有姓名的签条。"
+      },
+      "outputs": [{
+        "sentence": "舒双双给自己所有的衬衣都缝上了标有姓名的签条。"
+      }]
+    },
+    {
+      "class": "ChinesePersonNamedEntitiesAndGender",
+      "inputs": {
+        "sentence": "这些都是按照作者史永莉的字母顺序归档的。"
+      },
+      "outputs": [{
+        "sentence": "这些都是按照作者王克礼的字母顺序归档的。"
+      }]
+    },
+    {
+      "class": "ChinesePersonNamedEntitiesAndGender",
+      "inputs": {
+        "sentence": "我的名字是陈镇奎，我的年龄是20岁。我在江西武宁，我的工作是前台。",
+        "seed": 42
+      },
+      "outputs": [{
+        "sentence": "我的名字是曾祥莉，我的年龄是20岁。我在江西李洁修，我的工作是前台。"
```

Comment:
In this case, "武宁" should be a location, but being replaced with a new name. In the code, it simply searches for all occurrences of names in the dictionary and replaces them with new ones. 

I think a more decent way is to first identify whether the words are named entities (in this case, "武宁" should be excluded), and then do the replacement.

Reply:
Hi @Nickeilf,

Thanks for your feedback as always. :) I shall look into named entity recognition so that I'll be able to identify which ones are locations, names, or other entities before performing the replacement.

Thanks,
Tim

Reply:
Addressed